### PR TITLE
Fixes and improvements to hopper example

### PR DESCRIPTION
**Pull request by tkuchida**
Moved patella WrapCylinder back to thigh because of a bug in addToSystem() (see #1073). [ci skip]

### CODE REVIEW

**Comment by chrisdembia** [comment]
LGTM.

File: OpenSim/Sandbox/ExampleHopperDevice/buildHopperModel.cpp [hunk 1]
```diff
@@ -88,17 +88,17 @@ Model buildHopper() {
 
     // Set the coordinate names and default values. Note that we need "auto&"
     // here so that we get a reference to the CoordinateSet rather than a copy.
-    auto& sliderCoordSet = sliderToGround->upd_CoordinateSet();
-    sliderCoordSet[0].setName("height");
-    sliderCoordSet[0].setDefaultValue(1.);
+    auto& sliderCoord = sliderToGround->upd_CoordinateSet()[0];
+    sliderCoord.setName("height");
```
Comment (aseth1) [inline OpenSim/Sandbox/ExampleHopperDevice/buildHopperModel.cpp]: :+1:

**Comment by tkuchida** [comment]
This PR is isolated to the hopper example and repairs a bug on master, so I'm going to merge.

**Comment by chrisdembia** [comment]
🎆

**Pull request merged**